src/components/data/viewers/Toolbar.js

```diff
@@ -125,6 +134,20 @@ function ViewerToolbar(props) {
     return (
         <>
             <Toolbar variant="dense" id={baseId}>
+                <Button
+                    color="primary"
+                    variant={isMobile ? "text" : "contained"}
+                    size="small"
+                    startIcon={<ArrowBack fontSize="small" />}
+                    onClick={() => router.back()}
+                >
+                    <Hidden xsDown>{t("back")}</Hidden>
+                </Button>
+                <Divider
+                    orientation="vertical"
+                    flexItem
+                    className={classes.separator}
+                />
```

Comment:
While testing, the absence of `Back` bothered me so much, I added one right away :)